helm_deploy/secrets-example.yaml

```diff
@@ -2,3 +2,4 @@
 secrets:
   INTERVENTIONS_SERVICE_CLIENT_ID: clientid
   INTERVENTIONS_SERVICE_CLIENT_SECRET: clientsecret
+  APPINSIGHTS_INSTRUMENTATIONKEY: somesecretvalue
```

Comment:
The helm secret definition creates an environment variable called `APPLICATION_INSIGHTS_IKEY`, but this one is called `APPINSIGHTS_INSTRUMENTATIONKEY` -- which one should be used?

Reply:
Nice catch. I initially just reimplemented the code that was removed a while ago.
I've updated to match the variable within _envs.tpl